test/ui/common/attributes/AttributeEnumMapSettings.test.js

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 
 import 'test/enzyme-init';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 import AttributeEnumMapSettings, { elements as elementValidation } from 'ui/common/attributes/AttributeEnumMapSettings';
 import RenderTextInput from 'ui/common/form/RenderTextInput';
 import RenderSelectInput from 'ui/common/form/RenderSelectInput';
 import { required } from '@entando/utils';
+import { mockRenderWithIntl } from 'test/testUtils';
```

Comment:
please rename the file to understand it's using redux as well as intl on top of the mock